soc/microchip/pic32cmmc/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 # Copyright (c) 2025 GP Orcullo
 # SPDX-License-Identifier: Apache-2.0
 
+zephyr_sources(../../atmel/sam0/common/soc_port.c)
```

Comment:
create common directory for both vendors?

Reply:
I'm not sure on how to handle this one as Atmel is owned by Microchip. I don't know how they will consolidate this.